site/gdocs/OwidGdocPage.tsx

```diff
@@ -138,7 +139,30 @@ export default function OwidGdocPage({
                 <script
                     dangerouslySetInnerHTML={{
                         __html: `window._OWID_GDOC_PROPS = ${JSON.stringify(
-                            gdoc
+                            pick(gdoc, [
+                                "id",
+                                "slug",
+                                "content",
+                                "createdAt",
+                                "published",
+                                "publishedAt",
+                                "revisionId",
+                                "publicationContext",
+                                "breadcrumbs",
+                                "manualBreadcrumbs",
+                                "homepageMetadata",
+                                "latestDataInsights",
+                                "latestWorkLinks",
+                                "linkedAuthors",
+                                "linkedDocuments",
+                                "linkedCharts",
+                                "linkedIndicators",
+                                "imageMetadata",
+                                "relatedCharts",
+                                "tags",
+                                "donors",
+                                "relatedCharts",
+                            ])
```

Comment:
good optimization idea! Maybe we could use something similar to what's being done in `checkIsLightningUpdate()` so that we get some kind of typechecking and documentation about what's included/ommitted and why. https://github.com/owid/owid-grapher/blob/8bb950b3e78662cc9fc59acfb99b9d22000858b2/adminSiteClient/gdocsDeploy.ts#L25